### DISCUSSION TITLE
Make user and repo separate links on left sidebar

### DISCUSSION THREAD

**Discussion by madprops** [community/community#114039] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

On the main sidebar at the left. With the recent repos. It's a single link. Instead it could be separated into user and repo. Which would allow more flexibility of where you want to go.

Also make the recent repos to be way more sensitive. when I push to a repo I want it to be up there because it's actually relevant. Thanks :L

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @madprops , we're happy you're here! You are more likely to get a useful response if you are posting your feedback in the applicable category. I've gone ahead and moved it for you. Good luck!
